src/pages/publish-online-guidence/snippets.ts

```diff
@@ -0,0 +1,50 @@
+export const importCSSSnippet =`<link rel="stylesheet" type="text/css" href="https://cdn.statically.io/gh/digital-land/local-plans-timetable/master/timetable-visualisation.min.css"/>`;
+
+export const importJavaScriptSnippet =`<script src="https://cdn.statically.io/gh/digital-land/local-plans-timetable/master/timetable-visualisation.min.js"></script>`;
+
+export const renderTimetableVisualisationSnippet = `<script>
+  window.DLUHC.renderTimetableVisualisation(
+    {
+      developmentPlanFilepath:
+        "https://raw.githubusercontent.com/digital-land/local-plans-timetable/main/assets/development-plan.csv",
+      timetableEventsFilepath:
+        "https://raw.githubusercontent.com/digital-land/local-plans-timetable/main/assets/timetable.csv",
+    },
+    document.getElementById("timetable-visualisation")
+  );
+</script>`;
+
+export const exampleHTMLSnippet = `<html>
```

Comment:
Sorry to be really picky but if we move here and line 5 to a new line, our snippet matches exactly what we're rendering, e.g.

```ts
export const renderTimetableVisualisationSnippet = `
<script>
  window.DLUHC.renderTimetableVisualisation(
    {
      developmentPlanFilepath:
        "https://raw.githubusercontent.com/digital-land/local-plans-timetable/main/assets/development-plan.csv",
      timetableEventsFilepath:
        "https://raw.githubusercontent.com/digital-land/local-plans-timetable/main/assets/timetable.csv",
    },
    document.getElementById("timetable-visualisation")
  );
</script>
`;
```

Reply:
But this is such a small thing, I'm not too bothered if you don't want to do it 😆

Reply:
Initially I had those on a new line but then I have removed the new line because it was displaying some extra space on the top: 
![image](https://github.com/digital-land/local-plans-timetable/assets/98337074/8f602177-1283-4d6e-88fb-967407fed663)
![image](https://github.com/digital-land/local-plans-timetable/assets/98337074/13334d31-2f79-4b74-8798-58be2a9cab34)